server/src/entity/User.ts

```diff
@@ -0,0 +1,68 @@
+import { Entity, PrimaryGeneratedColumn, Column } from "typeorm"
+
+@Entity()
+export class User {
+
+    @PrimaryGeneratedColumn()
+    id: number
```

Comment:
This should be a type of UUID.

Reply:
Also, the primary key should be the `user_uid`

Reply:
Okay

Reply:
But user_id column sent in the table, its var char data type

Reply:
UUIDs are varchar.
So yea.

the user_uid is meant to be the primary key and not a separate column on the database. That id is coming from the frontend when the user authenticates with Pi

Reply:
Oh okay